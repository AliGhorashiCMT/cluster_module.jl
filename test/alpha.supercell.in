ion-species GBRV/$ID_pbesol.uspp 
coords-type Lattice
ion	O	0.2065	0.1334	0.107	1	
ion	O	0.7065	0.1334	0.107	1	
ion	O	0.2065	0.6334	0.107	1	
ion	O	0.7065	0.6334	0.107	1	
ion	O	0.2065	0.1334	0.607	1	
ion	O	0.7065	0.1334	0.607	1	
ion	O	0.2065	0.6334	0.607	1	
ion	O	0.7065	0.6334	0.607	1	
ion	O	0.3666	0.0731	0.2736665	1	
ion	O	0.8666	0.0731	0.2736665	1	
ion	O	0.3666	0.5730999999999999	0.2736665	1	
ion	O	0.8666	0.5730999999999999	0.2736665	1	
ion	O	0.3666	0.0731	0.7736665	1	
ion	O	0.8666	0.0731	0.7736665	1	
ion	O	0.3666	0.5730999999999999	0.7736665	1	
ion	O	0.8666	0.5730999999999999	0.7736665	1	
ion	O	0.4269	0.2935	0.4403335	1	
ion	O	0.9269000000000001	0.2935	0.4403335	1	
ion	O	0.4269	0.7935	0.4403335	1	
ion	O	0.9269000000000001	0.7935	0.4403335	1	
ion	O	0.4269	0.2935	0.9403334999999999	1	
ion	O	0.9269000000000001	0.2935	0.9403334999999999	1	
ion	O	0.4269	0.7935	0.9403334999999999	1	
ion	O	0.9269000000000001	0.7935	0.9403334999999999	1	
ion	O	0.1334	0.2065	0.393	1	
ion	O	0.6334	0.2065	0.393	1	
ion	O	0.1334	0.7065	0.393	1	
ion	O	0.6334	0.7065	0.393	1	
ion	O	0.1334	0.2065	0.893	1	
ion	O	0.6334	0.2065	0.893	1	
ion	O	0.1334	0.7065	0.893	1	
ion	O	0.6334	0.7065	0.893	1	
ion	O	0.0731	0.3666	0.2263335	1	
ion	O	0.5730999999999999	0.3666	0.2263335	1	
ion	O	0.0731	0.8666	0.2263335	1	
ion	O	0.5730999999999999	0.8666	0.2263335	1	
ion	O	0.0731	0.3666	0.7263335	1	
ion	O	0.5730999999999999	0.3666	0.7263335	1	
ion	O	0.0731	0.8666	0.7263335	1	
ion	O	0.5730999999999999	0.8666	0.7263335	1	
ion	O	0.2935	0.4269	0.0596665	1	
ion	O	0.7935	0.4269	0.0596665	1	
ion	O	0.2935	0.9269000000000001	0.0596665	1	
ion	O	0.7935	0.9269000000000001	0.0596665	1	
ion	O	0.2935	0.4269	0.5596665	1	
ion	O	0.7935	0.4269	0.5596665	1	
ion	O	0.2935	0.9269000000000001	0.5596665	1	
ion	O	0.7935	0.9269000000000001	0.5596665	1	
ion	Si	0.23495	0.0	0.1666665	1	
ion	Si	0.73495	0.0	0.1666665	1	
ion	Si	0.23495	0.5	0.1666665	1	
ion	Si	0.73495	0.5	0.1666665	1	
ion	Si	0.23495	0.0	0.6666665	1	
ion	Si	0.73495	0.0	0.6666665	1	
ion	Si	0.23495	0.5	0.6666665	1	
ion	Si	0.73495	0.5	0.6666665	1	
ion	Si	0.0	0.23495	0.3333335	1	
ion	Si	0.5	0.23495	0.3333335	1	
ion	Si	0.0	0.73495	0.3333335	1	
ion	Si	0.5	0.73495	0.3333335	1	
ion	Si	0.0	0.23495	0.8333335	1	
ion	Si	0.5	0.23495	0.8333335	1	
ion	Si	0.0	0.73495	0.8333335	1	
ion	Si	0.5	0.73495	0.8333335	1	
ion	Si	0.26505	0.26505	0.0	1	
ion	Si	0.76505	0.26505	0.0	1	
ion	Si	0.26505	0.76505	0.0	1	
ion	Si	0.76505	0.76505	0.0	1	
ion	Si	0.26505	0.26505	0.5	1	
ion	Si	0.76505	0.26505	0.5	1	
ion	Si	0.26505	0.76505	0.5	1	
ion	Si	0.76505	0.76505	0.5	1	


lattice\ 
 9.828 9.828 0.0 \ 
 -17.02259533678692 17.02259533678692 0.0 \ 
 0.0 0.0 21.62399999995872
