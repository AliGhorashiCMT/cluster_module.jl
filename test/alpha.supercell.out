
*************** JDFTx 1.6.0  ***************

Start date and time: Sat Jun  5 21:55:31 2021
Executable jdftx with command-line: -ni alpha.supercell.in
Running on hosts (process indices):  ALIs-MacBook-Pro.local (0)
Divided in process groups (process indices):  0 (0)
Resource initialization completed at t[s]:      0.00
Run totals: 1 processes, 8 threads, 0 GPUs


Input parsed successfully to the following command list (including defaults):

basis kpoint-dependent
coords-type Lattice
core-overlap-check vector
coulomb-interaction Periodic
davidson-band-ratio 1.1
dump End State
dump-name $INPUT.$VAR
elec-cutoff 20
elec-eigen-algo Davidson
elec-ex-corr gga-PBE
electronic-minimize  \
	dirUpdateScheme      FletcherReeves \
	linminMethod         DirUpdateRecommended \
	nIterations          100 \
	history              15 \
	knormThreshold       0 \
	energyDiffThreshold  1e-08 \
	nEnergyDiff          2 \
	alphaTstart          1 \
	alphaTmin            1e-10 \
	updateTestStepSize   yes \
	alphaTreduceFactor   0.1 \
	alphaTincreaseFactor 3 \
	nAlphaAdjustMax      3 \
	wolfeEnergy          0.0001 \
	wolfeGradient        0.9 \
	fdTest               no
exchange-regularization WignerSeitzTruncated
fluid None
fluid-ex-corr  lda-PZ
fluid-gummel-loop 10 1.000000e-05
fluid-minimize  \
	dirUpdateScheme      PolakRibiere \
	linminMethod         DirUpdateRecommended \
	nIterations          100 \
	history              15 \
	knormThreshold       0 \
	energyDiffThreshold  0 \
	nEnergyDiff          2 \
	alphaTstart          1 \
	alphaTmin            1e-10 \
	updateTestStepSize   yes \
	alphaTreduceFactor   0.1 \
	alphaTincreaseFactor 3 \
	nAlphaAdjustMax      3 \
	wolfeEnergy          0.0001 \
	wolfeGradient        0.9 \
	fdTest               no
fluid-solvent H2O 55.338 ScalarEOS \
	epsBulk 78.4 \
	pMol 0.92466 \
	epsInf 1.77 \
	Pvap 1.06736e-10 \
	sigmaBulk 4.62e-05 \
	Rvdw 2.61727 \
	Res 1.42 \
	tauNuc 343133 \
	poleEl 15 7 1
forces-output-coords Positions
ion O   0.206500000000000   0.133400000000000   0.107000000000000 1
ion O   0.706500000000000   0.133400000000000   0.107000000000000 1
ion O   0.206500000000000   0.633400000000000   0.107000000000000 1
ion O   0.706500000000000   0.633400000000000   0.107000000000000 1
ion O   0.206500000000000   0.133400000000000   0.607000000000000 1
ion O   0.706500000000000   0.133400000000000   0.607000000000000 1
ion O   0.206500000000000   0.633400000000000   0.607000000000000 1
ion O   0.706500000000000   0.633400000000000   0.607000000000000 1
ion O   0.366600000000000   0.073100000000000   0.273666500000000 1
ion O   0.866600000000000   0.073100000000000   0.273666500000000 1
ion O   0.366600000000000   0.573100000000000   0.273666500000000 1
ion O   0.866600000000000   0.573100000000000   0.273666500000000 1
ion O   0.366600000000000   0.073100000000000   0.773666500000000 1
ion O   0.866600000000000   0.073100000000000   0.773666500000000 1
ion O   0.366600000000000   0.573100000000000   0.773666500000000 1
ion O   0.866600000000000   0.573100000000000   0.773666500000000 1
ion O   0.426900000000000   0.293500000000000   0.440333500000000 1
ion O   0.926900000000000   0.293500000000000   0.440333500000000 1
ion O   0.426900000000000   0.793500000000000   0.440333500000000 1
ion O   0.926900000000000   0.793500000000000   0.440333500000000 1
ion O   0.426900000000000   0.293500000000000   0.940333500000000 1
ion O   0.926900000000000   0.293500000000000   0.940333500000000 1
ion O   0.426900000000000   0.793500000000000   0.940333500000000 1
ion O   0.926900000000000   0.793500000000000   0.940333500000000 1
ion O   0.133400000000000   0.206500000000000   0.393000000000000 1
ion O   0.633400000000000   0.206500000000000   0.393000000000000 1
ion O   0.133400000000000   0.706500000000000   0.393000000000000 1
ion O   0.633400000000000   0.706500000000000   0.393000000000000 1
ion O   0.133400000000000   0.206500000000000   0.893000000000000 1
ion O   0.633400000000000   0.206500000000000   0.893000000000000 1
ion O   0.133400000000000   0.706500000000000   0.893000000000000 1
ion O   0.633400000000000   0.706500000000000   0.893000000000000 1
ion O   0.073100000000000   0.366600000000000   0.226333500000000 1
ion O   0.573100000000000   0.366600000000000   0.226333500000000 1
ion O   0.073100000000000   0.866600000000000   0.226333500000000 1
ion O   0.573100000000000   0.866600000000000   0.226333500000000 1
ion O   0.073100000000000   0.366600000000000   0.726333500000000 1
ion O   0.573100000000000   0.366600000000000   0.726333500000000 1
ion O   0.073100000000000   0.866600000000000   0.726333500000000 1
ion O   0.573100000000000   0.866600000000000   0.726333500000000 1
ion O   0.293500000000000   0.426900000000000   0.059666500000000 1
ion O   0.793500000000000   0.426900000000000   0.059666500000000 1
ion O   0.293500000000000   0.926900000000000   0.059666500000000 1
ion O   0.793500000000000   0.926900000000000   0.059666500000000 1
ion O   0.293500000000000   0.426900000000000   0.559666500000000 1
ion O   0.793500000000000   0.426900000000000   0.559666500000000 1
ion O   0.293500000000000   0.926900000000000   0.559666500000000 1
ion O   0.793500000000000   0.926900000000000   0.559666500000000 1
ion Si   0.234950000000000   0.000000000000000   0.166666500000000 1
ion Si   0.734950000000000   0.000000000000000   0.166666500000000 1
ion Si   0.234950000000000   0.500000000000000   0.166666500000000 1
ion Si   0.734950000000000   0.500000000000000   0.166666500000000 1
ion Si   0.234950000000000   0.000000000000000   0.666666500000000 1
ion Si   0.734950000000000   0.000000000000000   0.666666500000000 1
ion Si   0.234950000000000   0.500000000000000   0.666666500000000 1
ion Si   0.734950000000000   0.500000000000000   0.666666500000000 1
ion Si   0.000000000000000   0.234950000000000   0.333333500000000 1
ion Si   0.500000000000000   0.234950000000000   0.333333500000000 1
ion Si   0.000000000000000   0.734950000000000   0.333333500000000 1
ion Si   0.500000000000000   0.734950000000000   0.333333500000000 1
ion Si   0.000000000000000   0.234950000000000   0.833333500000000 1
ion Si   0.500000000000000   0.234950000000000   0.833333500000000 1
ion Si   0.000000000000000   0.734950000000000   0.833333500000000 1
ion Si   0.500000000000000   0.734950000000000   0.833333500000000 1
ion Si   0.265050000000000   0.265050000000000   0.000000000000000 1
ion Si   0.765050000000000   0.265050000000000   0.000000000000000 1
ion Si   0.265050000000000   0.765050000000000   0.000000000000000 1
ion Si   0.765050000000000   0.765050000000000   0.000000000000000 1
ion Si   0.265050000000000   0.265050000000000   0.500000000000000 1
ion Si   0.765050000000000   0.265050000000000   0.500000000000000 1
ion Si   0.265050000000000   0.765050000000000   0.500000000000000 1
ion Si   0.765050000000000   0.765050000000000   0.500000000000000 1
ion-species GBRV/$ID_pbesol.uspp
ion-width 0
ionic-minimize  \
	dirUpdateScheme      L-BFGS \
	linminMethod         DirUpdateRecommended \
	nIterations          0 \
	history              15 \
	knormThreshold       0.0001 \
	energyDiffThreshold  1e-06 \
	nEnergyDiff          2 \
	alphaTstart          1 \
	alphaTmin            1e-10 \
	updateTestStepSize   yes \
	alphaTreduceFactor   0.1 \
	alphaTincreaseFactor 3 \
	nAlphaAdjustMax      3 \
	wolfeEnergy          0.0001 \
	wolfeGradient        0.9 \
	fdTest               no
kpoint   0.000000000000   0.000000000000   0.000000000000  1.00000000000000
kpoint-folding 1 1 1 
latt-move-scale 1 1 1
latt-scale 1 1 1 
lattice  \
	   9.827999999999999    9.827999999999999    0.000000000000000  \
	 -17.022595336786921   17.022595336786921    0.000000000000000  \
	   0.000000000000000    0.000000000000000   21.623999999958720 
lattice-minimize  \
	dirUpdateScheme      L-BFGS \
	linminMethod         DirUpdateRecommended \
	nIterations          0 \
	history              15 \
	knormThreshold       0 \
	energyDiffThreshold  1e-06 \
	nEnergyDiff          2 \
	alphaTstart          1 \
	alphaTmin            1e-10 \
	updateTestStepSize   yes \
	alphaTreduceFactor   0.1 \
	alphaTincreaseFactor 3 \
	nAlphaAdjustMax      3 \
	wolfeEnergy          0.0001 \
	wolfeGradient        0.9 \
	fdTest               no
lcao-params -1 1e-06 0.001
pcm-variant GLSSA13
spintype no-spin
subspace-rotation-factor 1 yes
symmetries automatic
symmetry-threshold 0.0001



---------- Setting up symmetries ----------

Found 24 point-group symmetries of the bravais lattice
Found 48 space-group symmetries with basis
Applied RMS atom displacement 2.94265e-06 bohrs to make symmetries exact.

---------- Initializing the Grid ----------
R = 
[        9.828        9.828            0  ]
[     -17.0226      17.0226            0  ]
[            0            0       21.624  ]
unit cell volume = 7235.31
G =
[   0.319657  -0.184554          0  ]
[   0.319657   0.184554         -0  ]
[         -0          0   0.290565  ]
Minimum fftbox size, Smin = [  80  80  88  ]
Chosen fftbox size, S = [  80  80  90  ]

---------- Exchange Correlation functional ----------
Initalized PBE GGA exchange.
Initalized PBE GGA correlation.

---------- Setting up pseudopotentials ----------
Width of ionic core gaussian charges (only for fluid interactions / plotting) set to 0

Reading pseudopotential file '/Users/ali/Desktop/JDFTX/build-withlibxc/pseudopotentials/GBRV/o_pbesol.uspp':
  Title: O.  Created by USPP 7.3.6 on 3-2-2014
  Reference state energy: -15.849885.  6 valence electrons in orbitals:
    |200>  occupation: 2  eigenvalue: -0.873214
    |210>  occupation: 4  eigenvalue: -0.330058
  lMax: 2  lLocal: 2  QijEcut: 6
  5 projectors sampled on a log grid with 511 points:
    l: 0  eig: -0.873214  rCut: 1.25
    l: 0  eig: 0.000000  rCut: 1.25
    l: 1  eig: -0.330058  rCut: 1.25
    l: 1  eig: 0.000000  rCut: 1.25
    l: 2  eig: 1.000000  rCut: 1.25
  Partial core density with radius 0.7
  Transforming core density to a uniform radial grid of dG=0.02 with 1442 points.
  Transforming local potential to a uniform radial grid of dG=0.02 with 1442 points.
  Transforming nonlocal projectors to a uniform radial grid of dG=0.02 with 432 points.
  Transforming density augmentations to a uniform radial grid of dG=0.02 with 1442 points.
  Transforming atomic orbitals to a uniform radial grid of dG=0.02 with 432 points.
  Core radius for overlap checks: 1.25 bohrs.

Reading pseudopotential file '/Users/ali/Desktop/JDFTX/build-withlibxc/pseudopotentials/GBRV/si_pbesol.uspp':
  Title: Si.  Created by USPP 7.3.6 on 23-7-2013
  Reference state energy: -4.568560.  4 valence electrons in orbitals:
    |300>  occupation: 2  eigenvalue: -0.395743
    |310>  occupation: 2  eigenvalue: -0.149098
  lMax: 2  lLocal: 3  QijEcut: 5
  6 projectors sampled on a log grid with 627 points:
    l: 0  eig: -0.395743  rCut: 1.6
    l: 0  eig: 1.000000  rCut: 1.6
    l: 1  eig: -0.149098  rCut: 1.6
    l: 1  eig: 1.000000  rCut: 1.6
    l: 2  eig: -0.100000  rCut: 1.7
    l: 2  eig: 0.100000  rCut: 1.7
  Partial core density with radius 1.45
  Transforming core density to a uniform radial grid of dG=0.02 with 1442 points.
  Transforming local potential to a uniform radial grid of dG=0.02 with 1442 points.
  Transforming nonlocal projectors to a uniform radial grid of dG=0.02 with 432 points.
  Transforming density augmentations to a uniform radial grid of dG=0.02 with 1442 points.
  Transforming atomic orbitals to a uniform radial grid of dG=0.02 with 432 points.
  Core radius for overlap checks: 1.70 bohrs.

Initialized 2 species with 72 total atoms.

Folded 1 k-points by 1x1x1 to 1 k-points.

---------- Setting up k-points, bands, fillings ----------
No reducable k-points. 
Computing the number of bands and number of electrons
Calculating initial fillings.
nElectrons: 384.000000   nBands: 192   nStates: 1

----- Setting up reduced wavefunction bases (one per k-point) -----
average nbasis = 30979.000 , ideal nbasis = 30909.782

---------- Setting up ewald sum ----------
Optimum gaussian width for ewald sums = 3.783034 bohr.
Real space sum over 567 unit cells with max indices [  4  4  3  ]
Reciprocal space sum over 7581 terms with max indices [  9  9  10  ]

---------- Allocating electronic variables ----------
Skipped wave function initialization.


---- Citations for features of the code used in this run ----

   Software package:
      R. Sundararaman, K. Letchworth-Weaver, K.A. Schwarz, D. Gunceler, Y. Ozhabes and T.A. Arias, 'JDFTx: software for joint density-functional theory', SoftwareX 6, 278 (2017)

   gga-PBE exchange-correlation functional:
      J.P. Perdew, K. Burke and M. Ernzerhof, Phys. Rev. Lett. 77, 3865 (1996)

   Pseudopotentials:
      KF Garrity, JW Bennett, KM Rabe and D Vanderbilt, Comput. Mater. Sci. 81, 446 (2014)

   Total energy minimization:
      T.A. Arias, M.C. Payne and J.D. Joannopoulos, Phys. Rev. Lett. 69, 1077 (1992)

This list may not be complete. Please suggest additional citations or
report any other bugs at https://github.com/shankar1729/jdftx/issues

Dry run successful: commands are valid and initialization succeeded.
End date and time: Sat Jun  5 21:55:32 2021  (Duration: 0-0:00:01.82)
Done!
